options/options.go

```diff
@@ -7,6 +7,7 @@ type Options struct {
 	StateDir        string  `long:"statedir" description:"Directory where ingest state is stored" default:"."`
 	HighAvail       bool    `long:"highavail" description:"Enable high availability ingestion using DynamoDB"`
 	BackfillHr      int     `long:"backfill" description:"The number of hours to increase backfill of log ingestion to with max of 168 hours (1 week)" default:"1"`
+	EdgeMode        bool    `long:"edge_mode" description:"Ignore any parent trace id, if present, from a load balancer"`
```

Comment:
Should we have a default of `false` here?

Reply:
Good catch. Turns out that boolean flags automatically default to false and can't have an explicit default value! So we're good here.

Reply:
👍

Reply:
Maybe we could add that `false` is the default in the description?